com.vrcfury.vrcfury/Runtime/VF/Model/Feature.cs

```diff
@@ -815,4 +816,10 @@ public class DisableLayer {
         }
     }
 
+    [Serializable]
+    public class OnDemandSync : NewFeatureModel {
+        public List<string> parameters;

```

Comment:
Make a wrapped class for this `string`, and also give it an additional `public bool ResetMePlease2;` (see `DisableLayer` above as an example). This allows us to add per-parameter options in the future, AND fixes the unity bug where added properties copy the last one in the list (our list implementation walks the tree and reconstructs any new objects containing the `ResetMePlease2` flag).

Reply:
The feature doesn't have serialized fields anymore, after simplifying to not needing any configuration